crates/core/tedge_mapper/src/collectd/monitor.rs

```diff
@@ -27,6 +28,7 @@ const DEFAULT_MAXIMUM_MESSAGE_DELAY: u32 = 400; // Heuristic delay that should w
 const DEFAULT_MESSAGE_LEAP_LIMIT: u32 = 0;
 const DEFAULT_MQTT_SOURCE_TOPIC: &str = "collectd/#";
 const DEFAULT_MQTT_TARGET_TOPIC: &str = "tedge/measurements";
+const DEFAULT_COLLECTD_MAPPER: &str = "tedge-mapper-collectd";
```

Comment:
Why a `DEFAULT` prefix? There is no way to change this name.

```suggestion
const COLLECTD_MAPPER_NAME: &str = "tedge-mapper-collectd";
```

Reply:
fixed